packages/deployment/scripts/2-deploy-faucet/input/faucet.js

```diff
@@ -0,0 +1,66 @@
+const { bigExp } = require('@aragon/contract-helpers-test')
```

Comment:
Similar to the comment above on the protocol, do you think this fits better as an `index.js`?